### DISCUSSION TITLE
Rename footnote section

### DISCUSSION THREAD

**Discussion by m-guggenmos** [quarto-dev/quarto-cli#6345] [Q&A] [answered]
### Description

Currently, the footnote section in revealjs presentations has the default name "Footnotes" (at least when using `reference-location: document` as I do). Is it possible to change this name e.g. to adapt it to a particular language?

**Comment by cderv** [comment, marked as answer]
You should be able to find that information in our doc: 

Document language is here: https://quarto.org/docs/authoring/language.html

* Changing `lang` will change the name of the section based on translation files
* If you are not happy with the translation in your language, use Alternate: https://quarto.org/docs/authoring/language.html#alternate-language

> You can discover all of the language values that can be customized by referencing this file: https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language.yml.

So go look at the source  and find what you need
https://github.com/quarto-dev/quarto-cli/blob/167c748ca99ad3ec2f4bc5414f887d9352c0fbd1/src/resources/language/_language.yml#L17

**Reply by m-guggenmos** [reply]
Perfect. I missed this in the docs.
